src/image_surface.rs

```diff
@@ -148,6 +149,14 @@ impl<'a> fmt::Display for ImageSurfaceData<'a> {
     }
 }
 
+// Workaround for cairo not having a direct way to check if the surface is finished.
+// See: https://gitlab.freedesktop.org/cairo/cairo/-/issues/406
+fn is_finished(surface: &ImageSurface) -> bool {
```

Comment:
I don't what "finished" refers to here...

Reply:
`cairo_surface_finish()`, it's a concept of cairo surfaces

Reply:
Ok, thanks!